src/app/workflows/workflows.component.html

```diff
@@ -13,15 +13,33 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
+
 <app-header>
-  <div fxLayout="row" fxLayoutAlign="start center">
+  <div fxLayout fxLayoutAlign="start center" *ngIf="searchPage === false; else search">
+    <img src="../../assets/svg/search.svg" class="site-icons-small mr-3" />
+    <div fxLayout fxLayoutGap="1rem">
+      <a
+        *ngIf="(entryType$ | async) === EntryType.BioWorkflow; else serviceOrNotebook"
+        [routerLink]="['/search']"
+        [queryParams]="{ entryType: workflows }"
```

Comment:
I think `workflows` needs to be quoted -- my IDE is flagging it.

Reply:
Thanks!